projects/common/src/navigation/navigation.service.ts

```diff
@@ -144,6 +149,10 @@ export class NavigationService {
     };
   }
 
+  public getActiveNavigation(): Navigation | undefined {
```

Comment:
what is the role of undefined here?

Reply:
This is based off https://angular.io/api/router/Router#getCurrentNavigation - it returns the in-flight navigation if there is one, otherwise null. I'm switching the null over to undefined here to keep consistency with our codebase.